src/main/java/lotto/domain/PurchaseLottoGroup.java

```diff
@@ -1,16 +1,32 @@
 package lotto.domain;
 
+import java.util.ArrayList;
 import java.util.List;
 import java.util.stream.Collectors;
 import java.util.stream.LongStream;
 
 public class PurchaseLottoGroup {
 	private final List<Lotto> lottoGroup;
 
-	public PurchaseLottoGroup(long quantity) {
-		lottoGroup =LongStream.rangeClosed(1, quantity)
-			.mapToObj(num -> new Lotto())
-			.collect(Collectors.toList());
+	private PurchaseLottoGroup(List<Lotto> lottos) {
+		this.lottoGroup = lottos;
+	}
+
+	public static PurchaseLottoGroup create(List<Lotto> manualLottoGroup, long autoQuantity) {
+		validateLottoGroup(manualLottoGroup);
+
+		List<Lotto> lottoGroup = new ArrayList<>(manualLottoGroup);
+		lottoGroup.addAll(LongStream.rangeClosed(1, autoQuantity)
+			.mapToObj(num -> Lotto.createAuto())
+			.collect(Collectors.toList()));
+
+		return new PurchaseLottoGroup(lottoGroup);
+	}
+
+	private static void validateLottoGroup(List<Lotto> lottoGroup) {
+		if (lottoGroup == null) {
```

Comment:
```suggestion
		if (lottoGroup == null || lottoGroup.isEmpty()) {
```

빈 값에 대한 체크도 추가되면 좋을 것 같네요 😄